clash_lib/src/common/geodata/geodata_proto.rs

```diff
@@ -0,0 +1,117 @@
+// This file is @generated by prost-build. DON'T edit
```

Comment:
did you generate this manually? make it a build dependency or add some notes for how to generate this in the .proto file?

Reply:
it was generated by prost-build, but the CI failed because protoc is not installed, so it's easier to put the generated code here.

Reply:
alright makes sense.